Containerfile.dev

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: MIT-0
 
-FROM docker.io/rust:1.82.0-alpine3.20
+FROM docker.io/rust:1.83.0-alpine3.20
```

Comment:
As of creating this Pull Request, the image for this version has not yet been published to [Docker hub]. Hence, the ["
Dev / Container" job](https://github.com/ericcornelissen/rust-rm/actions/runs/12104662402/job/33748336356?pr=292#step:4:26) failed.